tests/unittests/utils/test_ptv3.py

```
@@ -0,0 +1,312 @@
+        path = str(tmp_path / "weights.safetensors")
+        save_file({"embedding.stem.conv.weight": torch.zeros(3)}, path)
+
+        out = _load_backbone_state_dict(path)
+        assert "embedding.stem.conv.weight" in out
+
+    def test_raises_when_no_model_path(self):
+        with pytest.raises(ValueError):
+            _load_backbone_state_dict(None)
+
+    def test_raises_when_empty_model_path(self):
+        with pytest.raises(ValueError):
```

Ruff (0.15.17)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)